platform/android/java/lib/src/org/godotengine/godot/utils/PermissionsUtil.java

```diff
@@ -36,6 +36,8 @@
 import android.content.pm.PermissionInfo;
 import android.os.Build;
 import android.support.v4.content.ContextCompat;
+import android.util.Log;
+
```

Comment:
`clang-format` doesn't like this it seems.

Reply:
Note that if we want to enforce this kind of separation, we can do it in `.clang-format` with `IncludeCategories`.

Reply:
Actually `IncludeCategories` doesn't affect Java, but there's `JavaImportGroups`.